test/unit/portal-target.spec.js

```diff
@@ -44,11 +46,13 @@ describe('PortalTarget', function () {
 
   it('renders a single element for single vNode with slim prop & single slot element', () => {
     const vNode = Object.freeze(generateVNode())
-    Vue.set(transports, 'target', {
-      from: 'target-portal',
-      to: 'target',
-      passengers: vNode,
-    })
+    Vue.set(transports, 'target', [
```

Comment:
This change is correct and necessary, but was not applied to all test cases where  Vue.set was used.